youtube_downloader/cli.py

```
@@ -20,16 +55,57 @@ def main():
         elif sys.argv[i] == '--quality' and i + 1 < len(sys.argv):
             quality = sys.argv[i + 1]
             i += 2
+        elif sys.argv[i] == '--proxy-file' and i + 1 < len(sys.argv):
+            proxy_file = sys.argv[i + 1]
+            i += 2
+        elif sys.argv[i] == '--proxy' and i + 1 < len(sys.argv):
+            proxy_url = sys.argv[i + 1]
+            i += 2
+        elif sys.argv[i] == '--proxy-type' and i + 1 < len(sys.argv):
+            proxy_type = sys.argv[i + 1]
```

Ruff (0.14.1)

Local variable `proxy_type` is assigned to but never used

Remove assignment to unused variable `proxy_type`

(F841)